src/components/GridItems/GridItems.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import CardItem from "../CardItem/CardItems";
+import "./GridItems.css";
+import  cardInfo  from "../../data/griditems";
```

Comment:
create an entry point index.js 
```suggestion
import  cardInfo  from "../../data/";
```